website/plugins/remark-snackplayer/src/index.js

```diff
@@ -100,52 +83,7 @@ function SnackPlayer() {
         // To embed.js script
         const snackPlayerEmbed = u('html', {
           value: dedent`
-          <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/prism-solarized-dark@1.0.1/prism-solarizeddark.css">         
-          <script type="text/javascript">
-          // From: 
-          (function() {
-            'use strict';
-            if (typeof document === 'undefined') {
-              // Not on browser
-              return;
-            }
-            document.addEventListener('DOMContentLoaded', init);
-
-            function init() {
-              var mobile = isMobile();
-
-              window.ExpoSnack && window.ExpoSnack.initialize();
-
-              var snackPlayerList = document.querySelectorAll('.snack-player');
-
-              // Either show interactive or static code block, depending on desktop or mobile
-              for (var i = 0; i < snackPlayerList.length; ++i) {
-                var snackPlayer = snackPlayerList[i];
-                var snackDesktopPlayer = snackPlayer.querySelectorAll(
-                  '.desktop-friendly-snack'
-                )[0];
-                var plainCodeExample = snackPlayer.querySelectorAll(
-                  '.mobile-friendly-snack'
-                )[0];
-
-                if (mobile) {
-                  snackDesktopPlayer.remove();
-                  plainCodeExample.style.display = 'block';
-                } else {
-                  plainCodeExample.remove();
-                }
-              }
-            }
-            
-              // Primitive mobile detection
-              function isMobile() {
-                return /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(
-                  navigator.userAgent
-                );
-              }
-              
-          })();
-        </script>
+          <script async src="https://snack.expo.io/embed.js"></script>
```

Comment:
I have removed this script today, it is unnecessary in the case of RN website, because it has been added globally in config: 
* https://github.com/react-native-website-migration/react-native-website/blob/dev/website/docusaurus.config.js#L9